includes/class-newspack-popups.php

```diff
@@ -407,8 +407,7 @@ public static function create_lightweight_api_config() {
 			return;
 		};
 		add_option( 'newspack_has_tried_to_create_lightweight_api_config', true );
-		$has_db_config_in_env = getenv( 'DB_USER' ) && getenv( 'DB_PASSWORD' ) && getenv( 'DB_NAME' ) && getenv( 'DB_HOST' );
```

Comment:
I'm finding `getenv( 'DB_USER' )` returns nothing but `getenv()['DB_USER']` properly returns the environment variable.